packages/main/src/DatePicker.ts

```diff
@@ -405,6 +405,9 @@ class DatePicker extends DateComponentBase implements IFormInputElement {
 	}
 
 	onResponsivePopoverBeforeOpen() {
+		if (this._calendar) {
```

Comment:
In the _calendar getter you've put ! that means the query returns element always. If it's correct this expression is redundant otherwise the getter returns wrong result